cardano-api/src/Cardano/Api/LocalChainSync.hs

```diff
@@ -17,15 +17,15 @@ import           Ouroboros.Network.Protocol.ChainSync.Client
 
 
 -- | Get the node's tip using the local chain sync protocol.
-getLocalTip :: LocalNodeConnectInfo mode block -> IO (Tip block)
+getLocalTip :: LocalNodeConnectInfo mode block -> ExceptT error IO (Tip block)
 getLocalTip connctInfo = do
-    resultVar <- newEmptyTMVarIO
-    connectToLocalNode
-      connctInfo
-      nullLocalNodeClientProtocols {
-        localChainSyncClient = Just (chainSyncGetCurrentTip resultVar)
-      }
-    atomically (takeTMVar resultVar)
+    resultVar <- liftIO newEmptyTMVarIO
+    liftIO $ connectToLocalNode
+               connctInfo
+               nullLocalNodeClientProtocols {
+                 localChainSyncClient = Just (chainSyncGetCurrentTip resultVar)
+               }
+    liftIO $ atomically (takeTMVar resultVar)
```

Comment:
Is this change necessary?

Reply:
Not if you can figure out how to satisfy GHC :slightly_smiling_face: